_aggregations/pipeline/percentiles-bucket.md

```diff
@@ -0,0 +1,282 @@
+
+
+## Example
+
+The following example creates a date histogram with a one-week interval from the OpenSearch Dashboards e-commerce sample data. The `sum` sub-aggregation adds up the `taxful_total_price` for each week. Finally, the `percentiles_bucket` aggregation calculates the percentile values for each week from these sums:
+
+```json
+POST /opensearch_dashboards_sample_data_ecommerce/_search
+{
+  "size": 0,
+  "aggs": {
+    "weekly_sales": {
+      "date_histogram": {
+        "field": "order_date",
+        "calendar_interval": "week"
+      },
+      "aggs": {
+        "total_price": {
+          "sum": {
+            "field": "taxful_total_price"
+          }
+        }
+      }
+    },
+    "percentiles_monthly_sales": {
+      "percentiles_bucket": {
+        "buckets_path": "weekly_sales>total_price"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Example response
+
+The aggregation returns the default percentile values for the weekly price totals:
+
+<details open markdown="block">
+
+```json
+{
+  "took": 4,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 4675,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": []
+  },
+  "aggregations": {
+    "weekly_sales": {
+      "buckets": [
+        {
+          "key_as_string": "2025-03-24T00:00:00.000Z",
+          "key": 1742774400000,
+          "doc_count": 582,
+          "total_price": {
+            "value": 41455.5390625
+          }
+        },
+        {
+          "key_as_string": "2025-03-31T00:00:00.000Z",
+          "key": 1743379200000,
+          "doc_count": 1048,
+          "total_price": {
+            "value": 79448.60546875
+          }
+        },
+        {
+          "key_as_string": "2025-04-07T00:00:00.000Z",
+          "key": 1743984000000,
+          "doc_count": 1048,
+          "total_price": {
+            "value": 78208.4296875
+          }
+        },
+        {
+          "key_as_string": "2025-04-14T00:00:00.000Z",
+          "key": 1744588800000,
+          "doc_count": 1073,
+          "total_price": {
+            "value": 81277.296875
+          }
+        },
+        {
+          "key_as_string": "2025-04-21T00:00:00.000Z",
+          "key": 1745193600000,
+          "doc_count": 924,
+          "total_price": {
+            "value": 70494.2578125
+          }
+        }
+      ]
+    },
+    "percentiles_monthly_sales": {
+      "values": {
+        "1.0": 41455.5390625,
+        "5.0": 41455.5390625,
+        "25.0": 70494.2578125,
+        "50.0": 78208.4296875,
+        "75.0": 79448.60546875,
+        "95.0": 81277.296875,
+        "99.0": 81277.296875
+      }
+    }
+  }
+}
+```
+</details>
+
+## Example: options
+
+The following example computes percentiles on the same data as the previous example, but with the following differences:
```

Comment:
An intro sentence to a code block must end in a colon. This sentence is an intro to the list and to the code block, so we either have to change the list to a paragraph and end the last sentence in a colon, or add another short intro sentence for the code block.

Reply:
```suggestion
The next example computes percentiles on the same data as the previous example, but with the following differences.
```